modules/imgcodecs/test/test_animation.cpp

```diff
@@ -649,48 +678,48 @@ TEST(Imgcodecs_APNG, imread_animation_16u)
     Mat img = imread(filename, IMREAD_UNCHANGED);
     ASSERT_FALSE(img.empty());
     EXPECT_TRUE(img.type() == CV_16UC4);
-    EXPECT_EQ(0,     img.at<ushort>(0, 0));
-    EXPECT_EQ(0,     img.at<ushort>(0, 1));
-    EXPECT_EQ(65280, img.at<ushort>(0, 2));
```

Comment:
Was it alpha blending overflow issue?

Reply:
the old values was for the first frame before. now the first frame is `still_image `in the test file.

Reply:
the test was comparing correct values before. and comparing correct values now.